pkg/k8/deployments/deployments.go

```diff
@@ -190,7 +175,60 @@ func GetCNDPod(c *kubernetes.Clientset, namespace, deploymentName, devContainer
 		time.Sleep(1 * time.Second)
 	}
 
-	return nil, fmt.Errorf("kubernetes is taking too long to create the cnd container. Please check for errors or try again")
+	return nil, fmt.Errorf("kubernetes is taking too long to create the cloud native environment. Please check for errors or try again")
+}
+
+// InitVolumeWithTarball initializes the remote volume with a local tarball
+func InitVolumeWithTarball(c *kubernetes.Clientset, config *rest.Config, namespace, podName, folder string) error {
+	copied := false
+	tries := 0
+	for tries < 30 && !copied {
```

Comment:
Could we move this loop to its own function? I think it would be easier to read and understand.